roles/mesos/tasks/follower.yml

```diff
@@ -78,6 +78,15 @@
   tags:
     - mesos
 
+- name: set follower resources
+  sudo: yes
+  copy:
+    dest: /etc/mesos-slave/resources
+    content: "{{ mesos_resources }}"
+  notify: restart mesos follower
```

Comment:
@kenjones-cisco yes, please. We were going to put cluster-facing services on 4k-5k and let 31k-32k be automatically assigned. Adding ports is the first step, there.